ui/page/dexclient/add_dex_modal.go

```diff
@@ -66,91 +96,222 @@ func (md *addDexModal) Dismiss() {
 }
 
 func (md *addDexModal) OnDismiss() {
-	md.dexServerAddress.Editor.SetText("")
+	md.appPassword.Editor.SetText("")
+}
+
+func (md *addDexModal) WithAppPassword(appPass string) *addDexModal {
+	md.defaultAppPass = appPass
+	return md
 }
 
 func (md *addDexModal) OnResume() {
-	md.dexServerAddress.Editor.Focus()
+	md.appPassword.Editor.Focus()
+
+	clickable := func() *decredmaterial.Clickable {
+		cl := md.Theme.NewClickable(true)
+		cl.Radius = decredmaterial.Radius(0)
+		return cl
+	}
+
+	// Initialize listExchangeWdg.
+	listServer := sliceSever(core.CertStore[md.Dexc().Core().Network()])
```

Comment:
Please rename `sliceSever` to `sortExchanges`.
```suggestion
	dexServers := sortExchanges(core.CertStore[md.Dexc().Core().Network()])
```